### ISSUE TITLE
Update for Beat Saber 1.6.0

### ISSUE THREAD

**Issue by thezeagar** [bigfoott/FullComboDisplay#1]
An update has finally come out that this great mod will no longer work with.  Is there any chance we could get an update?

**Comment by bigfoott** [comment]
Woah, this mod has worked this whole time? Had no idea lol.

I'll take a look, thanks for letting me know!

**Comment by thezeagar** [comment]
Yep. I've been pleasantly surprised with every update when I'd drop this one in and it would still somehow be working.

Thanks for looking when you have a chance. I know there's a built in indicator now, but it's not as good.

**Comment by bigfoott** [comment]
No worries! It's currently finals week so it may be a bit till I get a chance to look, but it will likely be an easy fix!
